docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -0,0 +1,101 @@
+# Using cURL to troubleshoot Zowe CLI
+
+When a REST API call fails on Zowe CLI, you can use cURL to troubleshoot.
+
+Run a command with cURL and compare its output with what is returned using Zowe CLI. This can help pinpoint whether the problem lies with z/OSMF or Zowe CLI, depending on which command returns an API error.
+
+## Understanding cURL commands
+
+A cURL command using the `https` protocol follows the basic syntax `curl <URL>`.
+
+Add cURL options to establish communication between Zowe CLI and z/OSMF, as in the following example command:
+
+```
+curl --location --request <API method> "https://<host>:<port>/<API>" --header "X-CSRF-ZOSMF-HEADER;" --insecure --user "<ID>:<PASSWORD>"
+```
+**NOTE:** Some terminals may require single quotes rather than double quotes.
+
+
+### `--location`
+
+Use `--location` to allow the server to redirect to a different URL, if needed. 
+
+When a different location exists and `--location` is not included in the command, the server responds with a 3XX status code.
+
+### `--request <API method>`
+
+Use `--request` to identify the API method and name. Not necessary when the API method is `GET`.
+
+- `<API method>`: Specifies the API method used in the request.
+
+### `https://<host>:<port>/<API>"`
+
+Indicates the protocol and URL.
+
+- `<host>`: Specifies the host name where the z/OSMF services are running.
+- `<port>`: Specifies the REST port number.
+- `<API>`: Specifies the name of the API used in the request.
+
+### `--header "X-CSRF-ZOSMF-HEADER;"`
+
+Required to establish communication with z/OSMF. Specifies that the client is sending a cross-site request to the REST interface.
+
+- `;`: Indicates that the header has no value. (A value is not required.)
+
+### `--insecure`
+
+Use `--insecure` with a trusted server that does not require verification before a data transfer.
+
+### `--user "<ID>:<PASSWORD>"`
+
+Required and displays as plain text. Also possible to [use an environment variable](../../user-guide/cli-using-using-environment-variables.md).
+
+- `<ID>`: Specifies the z/OSMF user identification.
+- `<PASSWORD>`: Specifies the user password for z/OSMF.
```

Comment:
We could add a note here:
> To be prompted for your password instead of displaying it in plain text, omit the password from the command and specify `--user "<ID>"`.